src/super_gradients/qat_from_recipe.py

```diff
@@ -14,7 +14,7 @@
 
 @hydra.main(config_path="recipes", version_base="1.2")
 def _main(cfg: DictConfig) -> None:
-    QATTrainer.train_from_config(cfg)
+    QATTrainer.quantize_from_config(cfg)
```

Comment:
Since it is deprecated I think we should already change this script to using `Trainer`

Reply:
fixed in latest commit.